<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workWidget</class>
 <widget class="QWidget" name="workWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>运维工作台</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="workLay" stretch="2,8">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="workWidgeTree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="workWidgetStack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="workStackUsersMangeWidget"/>
         <widget class="QWidget" name="workStackRolesMangeWidge"/>
         <widget class="QWidget" name="workStackListAppWidge"/>
         <widget class="QWidget" name="workStackHandleAppWidget"/>
         <widget class="QWidget" name="p3"/>
         <widget class="QWidget" name="page_3"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
